routes/api/assets.js

```diff
@@ -73,7 +73,7 @@ router.put('/:path([^?]+)', interceptors.requireLogin, (req, res) => {
   });
 });
 
-router.get('/:path([^?]+)', interceptors.requireLogin, async (req, res) => {
+router.get('/:path([^?]+)', async (req, res) => {
```

Comment:
The requireLogin interceptor checks if a user is logged in before executing the route handling code. Removing the interceptor from this route allows the asset urls to be returned for non-logged in users.